### DISCUSSION TITLE
Nylo nested route

### DISCUSSION THREAD

**Discussion by alecoud** [nylo-core/nylo#169] [Ideas]
My App need sub navigator inside the bottom sheet, can Nylo framework support nested routes like go_router? This will allow us to develop more robust Apps.

**Comment by agordn52** [comment]
Hey @alecoud,

This would be a nice feature! I'll try think of some concepts.

**Reply by alecoud** [reply]
Hello @agordn52, it would be very cool if can support in future version, many thanks!

**Reply by alecoud** [reply]
Hello @agordn52 , now does the Nylo V6 Navigation Hub support the nested routes inside the bottom sheet? I tried pushTo(NewWidget()) within bottom sheet in the home tab , it navigated to a full screen new page but not open the NewWidget inside the home tab bottom sheet.

**Reply by agordn52** [reply]
> Hello @agordn52, it would be very cool if can support in future version, many thanks!

Yeah, feel free to reach out on Twitter/IG and we can arrange a call sometime.

**Reply by agordn52** [reply]
> Hello @agordn52 , now does the Nylo V6 Navigation Hub support the nested routes inside the bottom sheet? I tried pushTo(NewWidget()) within bottom sheet in the home tab , it navigated to a full screen new page but not open the NewWidget inside the home tab bottom sheet.

`pushTo` should push the new widget on the current tab. I'd have to see your project to understand this better.
